dictionaries/en_US/src/en_US.txt

```diff
@@ -3893,7 +3893,7 @@ consenter
 consenters
 consignataries
 consistorial
-consitutional
+constitutional
```

Comment:
Should probably drop down the list a bit to keep the alpha sort

Reply:
Sounds good. Will take care of that.